### ISSUE TITLE
Rounded borders with color set renders incorrectly

### ISSUE THREAD

**Issue by matto1990** [ptmt/react-native-macos#112]
Adding this example to line 207 of `Examples/UIExample/js/ViewExample.js` cause incorrect rendering.

```
  {
    title: 'Circle with Border Radius coloured',
    render: function() {
      return (
        <View style={{backgroundColor: '#527FE4', borderWidth: 1, borderColor: '#bb0000', borderRadius: 30, width: 60, height: 60}} />
      );
    },
  },
```

MacOS:

![image](https://cloud.githubusercontent.com/assets/97068/18187465/acc09b20-70a1-11e6-8f97-4b5de3203629.png)

iOS with the same code as above for comparison:

![image](https://cloud.githubusercontent.com/assets/97068/18187476/bdfdd1a0-70a1-11e6-8ada-a5ff08e74076.png)

Removing the `borderColor` style causes the output to be correct:

![image](https://cloud.githubusercontent.com/assets/97068/18187487/daa3ad0c-70a1-11e6-97ce-f1768f2165a4.png)

**Comment by matto1990** [comment]
I'm not certain why it would be related, but the only difference I can see between the iOS and MacOS border code is the lack of "resizing" support for MacOS. If that's causing this issue, then this category might be of interest: https://github.com/heardrwt/RHAdditions/blob/master/RHAdditions/NSImage%2BRHResizableImageAdditions.h

**Comment by ptmt** [comment]
It's related to differences in clipping. At first, I checked if I set it manually.
This worked as expected with clipped borders:
```
 {
    title: 'Circle with Border Radius coloured',
    render: function() {
      return (
        <View style={{overflow: 'hidden', backgroundColor: '#527FE4', borderWidth: 1, borderColor: '#bb0000', borderRadius: 30, width: 60, height: 60}} />
      );
    },
  },
```
However, on RN ios even clipping is disabled, it still works for borders.

![image](https://cloud.githubusercontent.com/assets/1004115/18194766/b6ae18a2-70f0-11e6-9124-00fad950b0c5.png)

macOS:
![image](https://cloud.githubusercontent.com/assets/1004115/18194798/f329c754-70f0-11e6-9ee5-998bc46338ac.png)

So I found that I didn't set the drawToEdges property correctly (because in RN for macOS clipsToBound has been introduced only around 0.6.0). Fixed now in latest commit (not published on npm yet):

![image](https://cloud.githubusercontent.com/assets/1004115/18195474/3d196808-70f4-11e6-9ee1-b57c9810f2ed.png)
